catalog/glue/glue.go

```diff
@@ -69,7 +69,7 @@ var (
 
 func init() {
 	catalog.Register("glue", catalog.RegistrarFunc(func(_ string, props iceberg.Properties) (catalog.Catalog, error) {
-		awsConfig, err := toAwsConfig(props)
+		awsConfig, err := toAwsConfig(context.Background(), props)
```

Comment:
should we change the registry to accept a context as part of `catalog.RegistrarFunc`?